build.sbt

```diff
@@ -1,7 +1,7 @@
 import com.typesafe.sbt.pgp.PgpKeys
 import sbtrelease.ReleaseStateTransformations._
 
-val monixVersion = "2.3.0"
+val monixVersion = "3.0.0-M3"
```

Comment:
I think we should upgrade to just ```2.3.3``` and do seperate branch for ```3.0.x```

Reply:
agreed, thanks